testing/agent-exporter/src/main/java/io/opentelemetry/javaagent/testing/exporter/OtlpInMemoryMetricExporter.java

```diff
@@ -30,6 +32,11 @@ void reset() {
     collectedRequests.clear();
   }
 
+  @Override
+  public AggregationTemporality getAggregationTemporality(InstrumentType instrumentType) {
```

Comment:
/cc @jack-berg Hopefully people will generally be able to use our metric exporters even in testing, but if for some reason someone has to implement this method they'll have absolutely no idea how to do so I think. I have a feeling defaulting to cumulative would have a net positive effect on confused users (it will also cause confusion, but my hypothesis is less so)